### PR TITLE
chore: alchemy provider replaced by decentraland rpc provider

### PR DESCRIPTION
**Pull request by 1emu**

### CODE REVIEW

File: Pulumi.website-dao-dev.yml [hunk 1]
```diff
@@ -2,8 +2,7 @@ encryptionsalt: v1:5g6OY0hHq10=:v1:2v1dmNBoUC4Ls4eB:jG0ttHuD/beq4tREu887OY++Kvz2
 config:
   aws:region: us-east-1
   website-dao:DEBUG_ADDRESSES: 0x8cff6832174091dae86f0244e3fd92d4ced2fe07,0xcD15d83f42179b9A5B515eea0975f554444a9646,0xb0F847e61C502Fb82D758C515b3F914de42831D5,0x6Cd7694d30c10bdAB1E644FC1964043a95cEEa5F,0xBf363AeDd082Ddd8DB2D6457609B03f9ee74a2F1
-  website-dao:ALCHEMY_API_KEY:
-    secure: v1:PJMog6uSWkPWCM06:82HTfltFoJVfYqWduuyg3x5dzx/FYEOTEugkSZ2JElI4MELJuzUEJRiYGE44D4jV
+  website-dao:RPC_PROVIDER_URL= https://rpc.decentraland.org/
```
Comment (andyesp) [inline Pulumi.website-dao-dev.yml:5]: `:` instead of `=`

File: Pulumi.website-dao-prd.yml [hunk 2]
```diff
@@ -2,8 +2,7 @@ encryptionsalt: v1:9oCf235QAQ8=:v1:uRW+FkAVO8o424II:P+oGJS3OY1Krmv/3faRNcy6mGYpZ
 config:
   aws:region: us-east-1
   website-dao:DEBUG_ADDRESSES: 0x8cff6832174091dae86f0244e3fd92d4ced2fe07,0xcD15d83f42179b9A5B515eea0975f554444a9646,0xb0F847e61C502Fb82D758C515b3F914de42831D5,0x6Cd7694d30c10bdAB1E644FC1964043a95cEEa5F,0xBf363AeDd082Ddd8DB2D6457609B03f9ee74a2F1
-  website-dao:ALCHEMY_API_KEY:
-    secure: v1:uimb7D29xvMwX7o7:MI7gQiCko1ZOEVIhKDtRtun4uP7Si2XJxKwV+Uaqr89P9lXcSlsMGOt4sqmX7zSx
+  website-dao:RPC_PROVIDER_URL= https://rpc.decentraland.org/
```
Comment (andyesp) [inline Pulumi.website-dao-prd.yml:5]: `:` instead of `=`

**Review by andyesp — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -47,7 +47,6 @@ if you are running this project locally you only need to check the following env
 * `GATSBY_SNAPSHOT_SPACE`: a snapshot space where the proposal will be published
 * `SNAPSHOT_PRIVATE_KEY`, `GATSBY_SNAPSHOT_ADDRESS`: a pair address/key with permissions to publish at that snapshot space
 * `DISCOURSE_API_KEY`: the api key use to publish the proposals on the forum
-* `ALCHEMY_API_KEY`: an alchemy api key to check the voting power
```
Comment (andyesp) [inline README.md:50]: As the rpc provider url is necessary to create proposals I'd add it to the readme.

**Review by ncomerci — APPROVED** [review]
I agree with Andy's comments, other than that it's ok! 🫡

**Review by andyesp — APPROVED** [review]

**Pull request merged**